JCDispatch/src/main/java/lib/nameParsing.java

```diff
@@ -0,0 +1,155 @@
+package lib;
+
+import com.auth0.jwt.internal.org.apache.commons.lang3.text.WordUtils;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+public class nameParsing {
+
+    public static String formatText( Object obj ) {
+        if ( obj != null ) {
+            String name = obj.toString().toLowerCase().trim();
+            name = name.replace( ",", ", " );
+            if (name.contains( "-" )) {
+                name = name.replace( "-", "- " );
+                return WordUtils.capitalize( name.trim() ).replace( "-", "-" );
+            }
+            return WordUtils.capitalize( name.trim() );
+        }
+        return null;
+    }
+
+    public static String removeDigit( Object obj ) {
+        String name = obj.toString().trim();
```

Comment:
This could throw a `NullPointerException` if `obj` is null. Since you're trying to get `obj` as a String, please use `Parsers.getAsString()`.